### ISSUE TITLE
Releases?

### ISSUE THREAD

**Issue by defensivedepth** [magicsword-io/sigconverter.io#34]
Any possibility that you all could cut a release every so often? Im currently specifying a commit hash in my build process rather than pulling directly from main.

**Comment by nasbench** [comment]
Good idea and it make sense for offline use. Give me some time to make something.

It'll probably be containing both versions (beta and main) so that people can use both locally (for latest features and what not)
